### PR TITLE
HOMS-239 Remove mount configs from docker-compose

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ v1.6.0 [unreleased]
 -   [#197](https://github.com/latera/homs/pull/197) Use Camunda as BPMS in default docker-compose.
 -   [#195](https://github.com/latera/homs/pull/195) Upgrade React & ReactDOM to 15.6.2, migrate to higher order components from mixins.
 -   [#153](https://github.com/latera/homs/pull/153) Allow updating users encrypted password and salt.
+-   [#239](https://github.com/latera/homs/pull/202) Remove mount configs from docker-compose.
```
Comment (sugarfree1) [inline CHANGELOG.md]: and set postgres version.

File: README.md [hunk 2]
```diff
@@ -37,46 +37,35 @@ The prefered way to install HOMS is to use Docker
...
-9. Navigate to [Minio control panel](http://localhost:9000) and create a bucket with name equal to `MINIO_BUCKET_NAME` value from `.env` file.
-
-10. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
+8. Navigate to [Minio control panel](http://localhost:9000) and create a bucket with name equal to `MINIO_BUCKET_NAME` value from `.env` file.
 
-11. (Optional) If you want to use demo processes navigate to Camunda Admin and create user with `user@example.com` email.
+9. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
 
-12. Login at [HydraOMS](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+10. (Optional) If you want to use demo processes navigate to Camunda Admin and create user with `user@example.com` email.
```
Comment (sugarfree1) [inline README.md]: Step 9 is pretty much the same as this step.

File: README.md [hunk 3]
```diff
@@ -92,15 +81,15 @@ Follow the same steps as for [In production installation](#in-production), but w
   git clone https://github.com/latera/homs.git
   ```
 
-4. Create your own configs from samples:
+4.1. Create your own configs from samples:
```
Comment (sugarfree1) [inline README.md]: Why 4.1.?

File: README.md (hunk 2 above)
Comment (mxsmirnoff) [reply README.md]: Yes, but it's optional. I decided to leave it as it was before, but change number.

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**